modules/phase_field/src/actions/PolycrystalVariablesAction.C

```diff
@@ -75,6 +77,9 @@ PolycrystalVariablesAction::act()
 
       var_params.applySpecificParameters(_pars, {"order", "family"});
       var_params.set<std::vector<Real>>("scaling") = {_pars.get<Real>("scaling")};
+      if (isParamValid("block"))
+        var_params.set<std::vector<SubdomainName>>("block") =
+            getParam<std::vector<SubdomainName>>("block");
```

Comment:
Would this not work?

```suggestion
      var_params.applySpecificParameters(_pars, {"order", "family", "block"});
      var_params.set<std::vector<Real>>("scaling") = {_pars.get<Real>("scaling")};
```